src/pages/chainlink-functions/tutorials/api-use-secrets.md

```diff
@@ -85,7 +85,7 @@ Reading the output of the example above, you can note that the `BTC/USD` price i
 ### Request
 
 :::note[Reminder]
```

Comment:
Could we single-source the common environment variable setup steps and then tack on any unique steps per tutorial? That could be more trouble than it's worth if we anticipate the underlying tutorial structure will change soon, but I'm not sure about the timing.

Reply:
To clarify: if we single-source the setup steps, then we could probably insert them directly into the tutorial. This seems like a good future enhancement.

Reply:
This notice is only on a couple of tutorials. Let's look into single-sourcing as a followup.